index.html

```diff
@@ -30,41 +30,234 @@
+            <div class="bottom__title">
+              <h2 class="txt_h2">THE STORY BEHIND <br />THE SOUCE</h2>
+
+              <img
+                src="/assets/images/girl.png"
+                alt="girl and souce"
+                class="bottom__title-img"
+              />
+            </div>
+            <div class="bottom__title-right">
+              <p class="bottom__p">
+                HELLO, I’M LAUREN AND AS A LIFELONG ADVENTUROUS EATER, I’VE
+                TRIED A LOT OF CRAZY THINGS, BUT I NEVER THOUGHT HOT SAUCE IN
+                COFFEE WAS A SANE OPTION UNTIL A FRIEND DARED ME TO TRY IT.
+              </p>
+              <p class="bottom__p">AND IT SUCKED.</p>
+              <p class="bottom__p">
+                BUT I WONDERED IF THERE WAS A WAY TO ADD SPICE TO MY COFFEE.
+                WHEN THERE WAS NO EASY SOLUTION, UJJO WAS BORN.
+              </p>
+              <p class="bottom__p">
+                UJJO (OOO-JOE): THE WORD FOR “FIRE” IN KONKANI, SPOKEN ON THE
+                WEST COAST OF INDIA, IS AN HOMAGE AND A LOVE LETTER TO MY DAD,
+                AN INDIAN IMMIGRANT WHO INSTILLED A FIRE IN MY PALATE – AND MY
+                HEART – FROM THE VERY BEGINNING.
+              </p>
+            </div>
+          </div>
+          <div class="bottom__down">
+            <div class="animation_border">
+              <div class="animation">
+                <img
+                  src="/assets/images/pepper.svg"
+                  alt="pepper"
+                  class="img__pepper"
+                />
+                <p class="animation-txt">
+                  CONTACT FOR PARTNERSHIP OPPORTUNITIES
+                </p>
+
+                <img
+                  src="/assets/images/pepper.svg"
+                  alt="pepper"
+                  class="img__pepper"
+                />
+                <p class="animation-txt">DOWNLOAD OUR MEDIA KIT</p>
+                <img
+                  src="/assets/images/pepper.svg"
+                  alt="pepper"
+                  class="img__pepper"
+                />
+                <p class="animation-txt">
+                  CONTACT FOR PARTNERSHIP OPPORTUNITIES
+                </p>
+
+                <img
+                  src="/assets/images/pepper.svg"
+                  alt="pepper"
+                  class="img__pepper"
+                />
+              </div>
+            </div>
+            <div class="bottom__image">
               <img
-                src="/assets/images/sauce_hand.png"
-                alt="hand holding sauce"
-                class="img__hand-sauce"
+                src="/assets/images/art.svg"
+                alt="art"
+                class="bottom__down-img"
               />
             </div>
           </div>
         </div>
       </section>
     </main>
-    <footer></footer>
+    <footer class="footer">
+      <div class="container">
+        <ul>
+          <li><a href="#" class="footer__nav-item">PRIVACY</a></li>
```

Comment:
`footer__nav-item` для `li` хорошо бы подошел, но не для линки

```suggestion
         <li><a href="#" class="footer__nav-item-link">PRIVACY</a></li>
```